appveyor.yml

```diff
@@ -0,0 +1,35 @@
+version: ImperatorToCk3.{build}
+skip_tags: true
+image: Visual Studio 2019 Preview
+configuration: Release
+build_script:
+- cmd: >-
+    git submodule update --init --recursive
+
+    msbuild ImperatorToCK3.sln /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"
+
+#   msbuild Vic2ToHoI4Tests/Vic2ToHoI4Tests-Google.vcxproj /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll
+
+#test_script:
+#- ps: >-
+#    vstest.console /logger:Appveyor frontend\Frontend.Core.Tests\bin\Release\Frontend.Core.Tests.dll
+
+#    cd Release\Vic2ToHoI4
+
+#    ..\..\Vic2ToHoI4Tests\Release\Vic2ToHoI4Tests.exe --gtest_output=xml:tests.xml
+
+#    (new-object net.webclient).UploadFile("https://ci.appveyor.com/api/testresults/junit/$($env:APPVEYOR_JOB_ID)", (Resolve-Path .\tests.xml))
+
+artifacts:
+- path: Release
+  name: ImperatorToCk3
+deploy:
+- provider: GitHub
+  release: WindowsRelease
```

Comment:
Now that the converters have moved back towards regular releases, rename this the `DevelopmentRelease` or something of the sort.

Reply:
I renamed it in this file, is anything else needed?

Reply:
That should do it. We'll see for certain once this gets merged.